app/scripts/app.js

```diff
@@ -634,6 +634,24 @@ angular
   })
   .run(function($rootScope, APIService) {
     $rootScope.AEROGEAR_MOBILE_ENABLED = !!APIService.apiInfo({ resource: "mobileclients", group: "mobile.k8s.io" });
+    if($rootScope.AEROGEAR_MOBILE_ENABLED){
+      // Add 'Mobile' category and sub-categories to the Service Catalog UI
+      window.OPENSHIFT_CONSTANTS.SERVICE_CATALOG_CATEGORIES.splice(
+        OPENSHIFT_CONSTANTS.SERVICE_CATALOG_CATEGORIES.length,
+        0,
+      {
+        id: 'mobile',
+        label: 'Mobile',
+        subCategories: [
+          { id: 'apps', label: 'Apps', tags: ['mobile'], icon: 'fa fa-mobile' },
+         {
+           id: 'services',
+          label: 'Services',
+          tags: ['mobile-service'],
+          icon: 'fa fa-database'
+        }]
+      });
+    }
     Logger.info("AEROGEAR_MOBILE_ENABLED: " + $rootScope.AEROGEAR_MOBILE_ENABLED);
```

Comment:
I feel that this block might grow in the future, we probably should move it to separate file and call it from here if possible.